java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -142,4 +145,64 @@ public void select_test_gives_error() {
                 assertThrows(ExecutionException.class, () -> regularClient.select(-1).get());
         assertTrue(e.getCause() instanceof RequestException);
     }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_no_args_returns_error() {
+        var exception =
+                assertThrows(
+                        ExecutionException.class, () -> regularClient.configGet(new String[] {}).get());
+        assertTrue(exception.getCause() instanceof RequestException);
+        assertTrue(exception.getCause().getMessage().contains("wrong number of arguments"));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_wildcard() {
+        var data = regularClient.configGet(new String[] {"*file"}).get();
+        assertTrue(data.size() > 5);
+        assertTrue(data.containsKey("pidfile"));
+        assertTrue(data.containsKey("logfile"));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configGet_with_multiple_params() {
+        assumeTrue(REDIS_VERSION.feature() >= 7, "This feature added in redis 7");
+        var data = regularClient.configGet(new String[] {"pidfile", "logfile"}).get();
+        assertAll(
+                () -> assertEquals(2, data.size()),
+                () -> assertTrue(data.containsKey("pidfile")),
+                () -> assertTrue(data.containsKey("logfile")));
+    }
+
+    @Test
+    @SneakyThrows
+    public void configSet_with_unknown_parameter_returns_error() {
+        var exception =
+                assertThrows(
+                        ExecutionException.class,
+                        () -> regularClient.configSet(Map.of("puppy", "happy")).get());
+        assertTrue(exception.getCause() instanceof RequestException);
+        assertTrue(
+                exception
+                        .getCause()
+                        .getMessage()
+                        .contains(
+                                REDIS_VERSION.feature() >= 7 ? "Unknown option" : "Unsupported CONFIG parameter"));
```

Comment:
message can be changed, we can simply check exception type

Reply:
We will adjust tests if CI get a new version